_markets/de/buy-resell/products/_archive.mdx

```diff
@@ -1,11 +1,22 @@
-### POS Archiv
+We offer two solutions for archiving data, see their Product Features below and [find more product details here](https://fiskaltrust.de/produkte/).
+MeinFiskal is described with its [technical details here](https://docs.fiskaltrust.cloud/de/docs/posdealers/buy-resell/products/3rd-party/datev-meinfiskal).
 
-The POS Archive is a **cloud service** which provides a searchable, secure, and **audit-proof** database for Middleware receipts. Activating it for a CashBox will also enable its Middleware instance to regularly push its new receipts into the archive, where they'll be securely persisted in a revision-safe storage. Due to german regulations, the data is kept for **at least ten years**.
+| Product Features | DATEV MeinFiskal    | PosArchiv           |
+|:----------------------|:--------------------|:----------------------|
+|Included in the Carefree Bundle |Yes  |Yes
+|Available as single product |No (in progress)  |Yes
+|Setup for multiple PosOperators |No |Yes  |
+|Place of backup |Germany  |Germany
+|Duration of backup |ten years  |ten years
+|Backup content |DFKA-Files (Daily-Closing), Receipts sent by fiskaltrust.Middleware and fiskaly relevant documentation   |Receipts sent by fiskaltrust.Middleware
```

Comment:
What is a "fiskaly relevant documentation"?

Also, let's just go with "DFKA taxonomy data" and "Raw receipt data" here or something similar. Or use the phrasing from the presentation yesterday :)

Reply:
spelling problem, also there is no hint (anymore) at the MeinFiskal homepage: There was an option to add relevant information material like Menus, price lists - this is no longer to be found.